examples/flutter_gallery/lib/demo/material/cards_demo.dart

```diff
@@ -59,102 +79,260 @@ class TravelDestinationItem extends StatelessWidget {
+                      Opacity(
+                        opacity: _isSelected ? 1 : 0,
+                        child: const Align(
+                            alignment: Alignment.topRight,
+                            child: Padding(
+                              padding: EdgeInsets.all(4.0),
+                              child: Icon(Icons.check_circle, color: Colors.white,),
+                            )
                         ),
                       ),
-                    ),
-                  ],
+                    ],
+                  )
                 ),
               ),
-              // description and share/explore buttons
-              Expanded(
-                child: Padding(
-                  padding: const EdgeInsets.fromLTRB(16.0, 16.0, 16.0, 0.0),
-                  child: DefaultTextStyle(
-                    softWrap: false,
-                    overflow: TextOverflow.ellipsis,
-                    style: descriptionStyle,
-                    child: Column(
-                      crossAxisAlignment: CrossAxisAlignment.start,
-                      children: <Widget>[
-                        // three line description
-                        Padding(
-                          padding: const EdgeInsets.only(bottom: 8.0),
-                          child: Text(
-                            destination.description[0],
-                            style: descriptionStyle.copyWith(color: Colors.black54),
-                          ),
-                        ),
-                        Text(destination.description[1]),
-                        Text(destination.description[2]),
-                      ],
-                    ),
-                  ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class SectionTitle extends StatelessWidget {
+  const SectionTitle({
+    Key key,
+    this.title
+  }) : super(key: key);
+
+  final String title;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.fromLTRB(4, 4, 4, 12),
+      child: Align(
+        alignment: Alignment.centerLeft,
+        child: Text(title, style: Theme.of(context).textTheme.subhead,),
+      ),
+    );
+  }
+}
+
+class TravelDestinationContent extends StatelessWidget {
+  TravelDestinationContent({ Key key, @required this.destination })
+      : assert(destination != null && destination.isValid),
+        super(key: key);
+
+  final TravelDestination destination;
+
+  @override
+  Widget build(BuildContext context) {
+    final ThemeData theme = Theme.of(context);
+    final TextStyle titleStyle = theme.textTheme.headline.copyWith(color: Colors.white);
+    final TextStyle descriptionStyle = theme.textTheme.subhead;
+
+    // Use Ink.image in order for the ink ripple to appear over the image
+    final Widget image = destination.type == CardDemoType.static
+        ? Image.asset(destination.assetName, package: destination.assetPackage, fit: BoxFit.cover,)
```

Comment:
Remove trailing comma

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-a-trailing-comma-for-arguments-parameters-and-list-items-but-only-if-they-each-have-their-own-line

Reply:
done